GiEnJul/ClientApp/src/components/admin/AdminPage.tsx

```diff
@@ -15,6 +16,11 @@ function AdminPage() {
         <AdminMenu />
         <Suggested />
         <Completed />
```

Comment:
If you want to view the table, remove this comment. I am going to continue working on the component and place it somewhere logical.